docs/reference/minimum-requirements.rst

```diff
@@ -0,0 +1,8 @@
+Minimum Requirements
+====================
+
+Before getting started with DSS, make sure your system meets the following requirements:
+
+* Ubuntu 22.04
+* Snapcraft installed
```

Comment:
We dont need snapcraft to run dss snap ... the devices must have snaps but every ubuntu later than 22.04 has it

Reply:
Let's also add a note here after the list to tell users that they might need more resources, depending on the Notebook OCI Image that they would run